### DISCUSSION TITLE
Question: How do i get the cropped image like in your demo?

### DISCUSSION THREAD

**Discussion by gregg-cbs** [ValentinH/react-easy-crop#263] [Q&A] [answered]
Hello,

I was wondering how do you get the cropped image from react-easy-crop?
like you do in your first example when clicking "show result":
https://ricardo-ch.github.io/react-easy-crop/

There are no requests happening to a server and it looks like react-easy-crop does not provide a cropped version of the image so im wondering how you did it.

Thanks,
Gregg

**Comment by ValentinH** [comment, marked as answer]
We do it using a canvas in memory.
You can check [this sandbox](https://codesandbox.io/s/q8q1mnr01w) or even better, as the demo website is open source, [the code of the demo](https://github.com/ricardo-ch/react-easy-crop/blob/main/docs/src/components/Demo/index.tsx).
